sdk/storage/storage-blob-changefeed/src/Chunk.ts

```diff
@@ -0,0 +1,48 @@
+import { AvroReader } from '../../storage-internal-avro/src';
+import { BlobChangeFeedEvent } from "./models/BlobChangeFeedEvent";
+
+export class Chunk {
+  private readonly _avroReader: AvroReader;
+  private readonly _iter: AsyncIterableIterator<Object | null>;
+
+  private _blockOffset: number;
+  public get blockOffset(): number {
+    return this._blockOffset;
+  }
+
+  private _eventIndex: number;
+  public get eventIndex(): number {
+    return this._eventIndex;
+  }
+
+  constructor(
+    avroReader: AvroReader,
+    blockOffset: number,
+    eventIndex: number
+  ) {
+    this._avroReader = avroReader;
+    this._blockOffset = blockOffset;
```

Comment:
interesting, what's the usage for blockOffset and eventIndex passed in from parameter?

Reply:
For resuming within the appendBlob (avro file).